DAG-flush-cache.py

```diff
@@ -0,0 +1,60 @@
+from datetime import datetime, timedelta
+
+from airflow.models import DAG, Variable
+from airflow.operators.email_operator import EmailOperator
+from airflow.operators.python import PythonOperator
+from airflow.utils.dates import days_ago
+from dag_datalake_sirene.task_functions import flush_cache
+
+DAG_NAME = "flush-cache"
+EMAIL_LIST = Variable.get("EMAIL_LIST")
+ENV = Variable.get("ENV")
+REDIS_HOST = "redis"
+REDIS_PORT = "6379"
+REDIS_DB = "0"
+REDIS_PASSWORD = Variable.get("REDIS_PASSWORD")
+
+
+default_args = {
+    "depends_on_past": False,
+    "email": EMAIL_LIST,
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 1,
+    "retry_delay": timedelta(minutes=5),
+}
+
+with DAG(
+    dag_id=DAG_NAME,
+    default_args=default_args,
+    schedule_interval="0 23 10 * *",
+    start_date=days_ago(10),
```

Comment:
this config will not start DAG automatically. That's what we want right ?

Reply:
Yes, this is the safe plug.
I will add the cache flushing to the main `isert-elk` dag where it will be executed after the aio build.